### DISCUSSION TITLE
Questions on draw.io

### DISCUSSION THREAD

**Discussion by volkiswag** [jgraph/drawio#4368] [Q&A] [answered]
Hello draw.io team,

at our company we are currently using diagram tools like Visio to make diagrams and mind maps. As these are quite overloaded and expensive we are trying to implement a new solution for our employees.
Draw.io is already in use in confluence but now we want to make the web- and desktop-version available as a standard software. 
To be able to roll it out as a standard software we need to fill out a questionnaire. 
We are aware that draw.io is an open-source product and your time is probably very limited, but we would like to ask you a few questions on draw.io to be able to use it as a standard software.
To save time we shortened the questionnaire to the following questions that are still open.

1.	Please describe the systems architecture and the basic components of your platform. 

2.	Does your solution utilize cloud native services (>= PaaS, Microservices, etc.)?

3.	Who is responsible to bring 3rd party licenses? (e.g. Oracle, Java)

4.	Which transport protocols & encryption do you use?

5.	Which functionalities of your solution are accessible via API?

6.	Are all data assets within your solution accessible via API?

7.	Are there 3rd party partners involved / get access / consume data of us out of your solution (e.g. google analytics)?

8.	Is it possible for us to audit and export the workflow & event management logs?

9.	Does your solution provide the capability to monitor from an End-to-End perspective?

10.	Do you refer on ITIL v3/ 4 standard?

11.	Please describe your update process for minor and major updates.

**Comment by davidjgraph** [comment]
Most of the answers differ between this repo and the desktop repo. Please post again on the desktop repo with the question list adapted to a desktop app.

In terms of this repo, are you asking these with a view to deploying this repo internally, or using the reference deployment at app.diagrams.net?

**Reply by volkiswag** [reply]
Im not sure if I understand your question correctly but our plan is to make the desktop version and the online version under app.diagramm.net available for use.

**Comment by davidjgraph** [comment, marked as answer]
The answers for app.diagrams.net:



    Please describe the systems architecture and the basic components of your platform.

It's a mostly static JS application, delivered via Cloudflare edge nodes.

    Does your solution utilize cloud native services (>= PaaS, Microservices, etc.)?

It uses Cloudflare workers and AWS lambda for things like import, authenication, etc.

    Who is responsible to bring 3rd party licenses? (e.g. Oracle, Java)

Don't understand.

    Which transport protocols & encryption do you use?

It uses whatever the browser negotiates with Cloudflare endpoints. Cloudflare supports the most recent protocols, so it depends on the browser used.

    Which functionalities of your solution are accessible via API?

Nothing, we don't store any data.

    Are all data assets within your solution accessible via API?

N/A, we don't store any data.

    Are there 3rd party partners involved / get access / consume data of TRUMPF out of your solution (e.g. google analytics)?

No

    Is it possible for us to audit and export the workflow & event management logs?

No

    Does your solution provide the capability to monitor from an End-to-End perspective?

Specifically?

    Do you refer on ITIL v3/ 4 standard?

No

    Please describe your update process for minor and major updates.

We have a release process that is deemed compliant with SOC 2, we haven't completed an audit period yet.

**Reply by volkiswag** [reply]
Thanks a lot for your support.
